test/python/conftest.py

```diff
@@ -56,6 +56,8 @@ def pytest_generate_tests(metafunc):
 
 def prepare_speculos_args(firmware):
     speculos_args = ["--model", firmware.device, "--sdk", firmware.version]
```

Comment:
Not related to the PR, but as you change this file, I think this line is redundant [with `ragger` internal behavior](https://github.com/LedgerHQ/ragger/blob/develop/src/ragger/backend/speculos.py#L60) and could be removed.

Reply:
Tout à fait, on donne l'argument en double actuellement il semblerait